app/services/api/webflow-helpers.ts

```diff
@@ -118,17 +118,30 @@ export const cleanedWorkshops = (
  */
 const convertScheduleToCardProps = (schedule: ScheduledEvent): ScheduleCardProps => {
   switch (schedule.type) {
+    case "Sponsored":
+      return {
+        variant: "recurring",
+        formattedStartTime: formatDate(schedule["day-time"], "h:mm aaa"),
+        startTime: schedule["day-time"],
+        sources: [],
+        eventTitle: schedule.name,
+        subheading: schedule["event-description"],
+        id: schedule._id,
+      }
     case "Recurring":
+      // eslint-disable-next-line no-case-declarations
+      const isTheSameTime = schedule["day-time"] === schedule["end-time"]
       return {
         variant: "recurring",
         formattedStartTime: formatDate(
           schedule["day-time"],
-          schedule["end-time"] ? "h:mm" : "h:mm aaa",
+          schedule["end-time"] && !isTheSameTime ? "h:mm" : "h:mm aaa",
         ),
         startTime: schedule["day-time"],
-        formattedEndTime: schedule["end-time"] && formatDate(schedule["end-time"], "h:mm aaa"),
-        eventTitle: schedule["recurring-event"]?.name,
-        heading: "",
+        formattedEndTime:
+          schedule["end-time"] && !isTheSameTime && formatDate(schedule["end-time"], "h:mm aaa"),
```

Comment:
This makes me wonder if we shouldn't have a `formattedTime` attribute that automatically handles the same time and formats it appropriately.

Reply:
totally open if you'd like to branch off of this and handle that?